### PR TITLE
Uior 123 add related invoices accordion

### PR DESCRIPTION
**Pull request by Feboll**
## Purpose

https://issues.folio.org/browse/UIOR-123

## Approach

Fixed bugs with displaying related invoices

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.

- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [ ] Were any API paths or methods changed, added or removed?
  - [ ] Were there any schema changes?
  - [ ] Did any of the interface versions change?
  - [ ] Were permissions changed, added, or removed?
  - [ ] Are there new interface dependencies?
  - [ ] There are no breaking changes in this PR.
  
If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?  
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.  

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/components/POLine/POLineInvoices/POLineInvoicesContainer.js [hunk 1]
```diff
@@ -7,23 +7,34 @@ import { stripesConnect } from '@folio/stripes/core';
...
-  resources,
-  vendors,
-}) => {
+const POLineInvoicesContainer = ({ accordionId, label, resources, vendors, mutator }) => {
   const lineInvoices = get(resources, ['invoices', 'records'], []);
   const invoiceLines = get(resources, ['invoiceLines', 'records'], []);
   const pieces = get(resources, ['pieces', 'records'], []);
 
+  useEffect(() => {
+    mutator.invoiceLines.reset();
```
Comment (NikitaSedyx) [inline src/components/POLine/POLineInvoices/POLineInvoicesContainer.js]: don't you need to check `lineId` change in effect?

File: src/components/POLine/POLineInvoices/POLineInvoices.js [hunk 2]
```diff
@@ -51,6 +51,7 @@ const POLineInvoices = ({ lineInvoices, invoiceLines, vendors, pieces }) => {
     quantity: (invoice) => sum(invoiceLines.filter(invoiceLine => invoiceLine.invoiceId === invoice.id).map(inv => get(inv, 'quantity', 0))),
     expendedAmount: invoice => (
       <AmountWithCurrencyField
+        stripes={stripes}
```
Comment (NikitaSedyx) [inline src/components/POLine/POLineInvoices/POLineInvoices.js]: `AmountWithCurrencyField` is wrapped by `withStipes`

File: src/components/POLine/POLineInvoices/POLineInvoicesContainer.js [hunk 3]
```diff
@@ -43,6 +55,7 @@ POLineInvoicesContainer.propTypes = {
   // eslint-disable-next-line react/no-unused-prop-types
   lineId: PropTypes.string.isRequired,
   label: PropTypes.object.isRequired,
+  stripes: PropTypes.object.isRequired,
   accordionId: PropTypes.string.isRequired,
```
Comment (aliaksei-chumakou) [inline src/components/POLine/POLineInvoices/POLineInvoicesContainer.js]: Guys, maybe I'm missing something, but what is the reason of this prop, `accordionId` ? It looks to me that just a constant is passed down

File: src/components/POLine/POLineInvoices/POLineInvoicesContainer.js (hunk 1 above)
Comment (aliaksei-chumakou) [reply src/components/POLine/POLineInvoices/POLineInvoicesContainer.js]: I thought it goes as a second argument, `}, [lineId]);` line 36

File: src/components/POLine/POLineInvoices/POLineInvoicesContainer.js (hunk 3 above)
Comment (Feboll) [reply src/components/POLine/POLineInvoices/POLineInvoicesContainer.js]: You are right

File: src/components/POLine/POLineInvoices/POLineInvoicesContainer.js (hunk 1 above)
Comment (Feboll) [reply src/components/POLine/POLineInvoices/POLineInvoicesContainer.js]: I did this

**Pull request merged**